courses.html

```diff
@@ -0,0 +1,186 @@
+                            </div>
+                            <div class="buy-body-second-row">
+                                <div class="buy-page-text text-hardness-low">
+                                    <P>second year</P>
+                                </div>
+                                <div class="price-right-align">€600.00
+                                </div>
+                            </div>
+                             <div class="buy-body-third-row">
+                                <div class="buy-page-text text-hardness-low">
+                                    <a class="nav-links__link nav-links__link--color-red" href="index.html">Request demo</a>
+                                </div>
+                                <div class="price-right-align">
+                                    <a href="inndex.html" class="hero-btn hero-btn--red hero-btn--rounded">Buy</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+            </div>
+        </section>
+        <section class="footer">
+            <h4 class="footer__heading">About Us</h4>
+            <p class="footer__copy">Excellence University is a leading Educational institution based out of Delhi. The University has established itself as a high quality education provider<br> with prime focus on holistic learning and imbibing competitive abilities in students.</p>
+            <div class="footer_icons">
+                <i class="fab fa-facebook style-icon"></i>
+                <i class="fab fa-twitter style-icon"></i>
+                <i class="fab fa-instagram style-icon"></i>
+                <i class="fab fa-linkedin style-icon"></i>
+                    
+            </div>
+        </section>  
+        
+         <script>
+        const allMatchingElementsText= document.querySelectorAll(".buy-page-text .price-text");
+        const allMatchingElementsPrice= document.querySelectorAll(".price");
+        const secondYearPrice= document.querySelectorAll(".buy-body-second-row"); 
+        document.getElementById("plan-selection").addEventListener("click", changeContentAsPerSelectedPlan);
+        function changeContentAsPerSelectedPlan(){
+		console.log(event);	
+		var planType = event.target.getAttribute("data-plan");	
+		document.querySelectorAll(".selectButton").forEach((x) => {x.classList.remove("current-tab")});
+		event.target.classList.add("current-tab");
+		if(planType == "monthly") {
+            allMatchingElementsText.forEach((matchingElement) => {
+                 matchingElement.innerHTML = "per month";
+}); 
+            allMatchingElementsPrice[0].innerHTML = "€49.00";
+            allMatchingElementsPrice[1].innerHTML = "€59.00";
+            allMatchingElementsPrice[2].innerHTML = "€69.00";
+            secondYearPrice.forEach((matchingElement1) => {
+                 matchingElement1.style.display= "none";
+}); 
+}
+		else if(planType == "annual"){
```

Comment:
```suggestion
		else if (planType === "annual"){
```

There is a different between == and === operator, last one compares by type not only value and it's better to always use === to avoid unexpected compare results